include/TMU.hpp

```diff
@@ -58,7 +59,7 @@ class TMU {
     /**
```

Comment:
You need to declare the error array as a private variable here, so the object dictionary can reference it and output over CAN